### DISCUSSION TITLE
Newbie Here

### DISCUSSION THREAD

**Discussion by radiantone** [RobotWebTools/roslibjs#755] [General]
Hi,
  I was looking at the simple example here https://github.com/RobotWebTools/roslibjs/blob/develop/examples/simple.html
  And trying to understand where the service AddTwoInts is implemented? I don't see where the 2 numbers are added together and returned.
  
  Thank you!

**Comment by MatthijsBurgh** [comment]
This is done somewhere else. As you call a service to do it.
